apps/src/code-studio/pd/workshop_dashboard/components/organizer_form_part.jsx

```diff
@@ -12,12 +12,38 @@ const styles = {
 
 export default class OrganizerFormPart extends React.Component {
   static propTypes = {
-    potential_organizers: PropTypes.array,
-    organizer_id: PropTypes.number,
+    workshopId: PropTypes.number,
+    organizerId: PropTypes.number,
     onChange: PropTypes.func,
     readOnly: PropTypes.bool
   };
 
+  state = {
+    loading: true,
+    potentialOrganizers: null
+  };
+
+  componentWillMount() {
+    this.load(this.props.workshopId);
+  }
+
+  load() {
+    let url = `/api/v1/pd/workshops/${
+      this.props.workshopId
+    }/potential_organizers`;
+
+    $.ajax({
+      method: 'GET',
+      url: url,
+      dataType: 'json'
+    }).done(data => {
+      this.setState({
+        loading: false,
+        potentialOrganizers: data
+      });
```

Comment:
What happens in the case of an error, particularly in the user experience?

Reply:
The dropdown appears blank (with no options) but the rest of the page loads and works normally. Do you have suggestions for a better user experience?

Reply:
now in the event of an error, it displays the organizer name instead of the dropdown.

Reply:
Should the user know that an error has occurred?  I think it's not unreasonable to explain that something went slightly wrong, even if the only remedy is to reload the whole page.

Reply:
That makes sense. I added an error message - screenshot in PR description. thanks!